app/routes/application.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
 
 import OSFAgnosticAuthRouteMixin from 'ember-osf/mixins/osf-agnostic-auth-route';
+import AnalyticsMixin from '../mixins/analytics-mixin';
 
-export default Ember.Route.extend(OSFAgnosticAuthRouteMixin, {
+export default Ember.Route.extend(AnalyticsMixin, OSFAgnosticAuthRouteMixin, {
```

Comment:
It might be easier to reopen the `Route` class to force the `AnalyticsMixin` to be applied to every route.